site/pages/retirements/[beneficiary]/[retirement_index].tsx

```diff
@@ -84,6 +88,14 @@ export const getStaticProps: GetStaticProps<PageProps, Params> = async (
       beneficiaryAddress = beneficiaryInUrl;
     }
 
+    let pledge: Pledge | null = null;
+    try {
+      /** An undefined pledge is valid here */
+      pledge = await getPledgeByAddress(beneficiaryAddress);
```

Comment:
```suggestion
      pledge = await getPledgeByAddress(beneficiaryAddress.toLowerCase());
```

`ownerAddress` for pledges are saved to firebase in lowercase format (we'll be changing this in the future). 
Not sure if this is already handled in some helper utility fn used on this page tho.
Hopeful that this would correctly fetch the pledge 🤞